### DISCUSSION TITLE
Start mining with dogecoind or by dogecoin-cli ?

### DISCUSSION THREAD

**Discussion by itmagpro** [dogecoin/dogecoin#3658] [Q&A] [answered]
Why not anywhere in dogecoin doc no exist detailed info about start mining with dogecoind or by dogecoin-cli ? ;(

This is start mining cmd with dogecoin-cli ?

```
dogecoin-cli generate 666000000
or
dogecoin-cli generatetoaddress 666000000 WALLET_ADDRESS
```

The how we can specified generate unlimited num ?

dogecoin not support auto start mining without RPC command ?

**Comment by 0foxes** [comment, marked as answer]
This functionality is deprecated as most Dogecoin is merge mined with Litecoin using ASIC machines, and you will likely never successfully mine a block from your CPU. 

If you are trying to generate testnet coins, you can simply put a sufficiently large number. As Dogecoin's target block time is 1 minute, `dogecoin-cli generate 666000000` on the testnet would take 1200+ years to generate and is effectively a "try to generate unlimited blocks" command.

**Comment by itmagpro** [comment]
> you will likely never successfully mine a block from your CPU

### but if mining result always depend from random (or not?) hashes like in lottery, then, theoretically can be miracle ? :)

OR NOT?

## or dogecoin engine independent from random hashes and "yay" (not "boo") result accept only from miner's who have biggest ... ASIC hash rate?

**then this unfairly (unjust) and not like in lottery and seems to be like scam ;(** and lost lot of money for buy ASIC's, example $10 000, for mining $10 099 or even lower.., - also unprofitable and **wasted (stolen) work and time!**

I think, that all this cryptocurrency, this A BIG FAKE, and only author this destruct cryptocurrency project's know how bypass all underwater stones and get maximum profit from this BIG SCAM.

Then, more profitable (proof of works) may be only simple paper lottery or bank robbery.

# P.S. I think, that all cryptocurrency this a BIG FAKE AND BIG SCAM FOR DESTROY old FINANCIALLY SYSTEM IN WHOLE WORLD AND MOVED ALL PEOPLE IN DIGITAL JAIL with brain chip and etc...

In Elon Musk plans to do this with each people in the whole world! Proof of link:

[Microchip implant (human) - Wikipedia](https://en.wikipedia.org/wiki/Microchip_implant_(human))

or for Russian auditory

[Чипирование людей — Википедия](https://ru.wikipedia.org/wiki/Чипирование_людей)

_Just wait, wasted your self time on fake mining scam cryptocurrencies and etc stupid works.., and this came sun..._

**Comment by Jody78** [comment]
dogecoin-cli generate 666000000
or
dogecoin-cli generatetoaddress 666000000 WALLET_ADDRESS

**Comment by satoshi9000** [comment]
developers should definitely introduce mining into dogecoin core to avoid the centralization of mining and not have the world go to their heads
